diff --git a/node_modules/@2060.io/ffi-napi/src/ffi.cc b/node_modules/@2060.io/ffi-napi/src/ffi.cc
index abe468b..5f148a8 100644
--- a/node_modules/@2060.io/ffi-napi/src/ffi.cc
+++ b/node_modules/@2060.io/ffi-napi/src/ffi.cc
@@ -1,5 +1,5 @@
 #define NAPI_VERSION 6
-#define NAPI_EXPERIMENTAL /* Until Node.js v12.17.0 is released */
+
 #include "ffi.h"
 #include "fficonfig.h"
 #include <get-uv-event-loop-napi.h>
