diff --git a/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js b/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js
index c833d0108e8852127afac1c87cffca0aba1268dc..09e51d085958bacf4dc2e08985d6f48edbc47c54 100644
--- a/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js
+++ b/build/modules/vc/data-integrity/W3cJsonLdCredentialService.js
@@ -177,6 +177,7 @@ let W3cJsonLdCredentialService = class W3cJsonLdCredentialService {
             suite: suite,
             challenge: options.challenge,
             domain: options.domain,
+            purpose: options.proofPurpose,
             documentLoader: this.w3cCredentialsModuleConfig.documentLoader(agentContext),
         });
         return utils_1.JsonTransformer.fromJSON(result, W3cJsonLdVerifiablePresentation_1.W3cJsonLdVerifiablePresentation);
