diff --git a/node_modules/web-did-resolver/lib/resolver.cjs b/node_modules/web-did-resolver/lib/resolver.cjs
index 4ece603..2e511c0 100644
--- a/node_modules/web-did-resolver/lib/resolver.cjs
+++ b/node_modules/web-did-resolver/lib/resolver.cjs
@@ -203,7 +203,7 @@ function getResolver() {
       if (id.length > 1) {
         path = id.map(decodeURIComponent).join('/') + '/did.json';
       }
-      const url = `https://${path}`;
+      const url = `http://${path}`;
       const didDocumentMetadata = {};
       let didDocument = null;
       const _temp3 = _do(function () {
